management/src/main/java/io/micronaut/management/health/aggregator/DefaultHealthAggregator.java

```diff
@@ -112,9 +116,21 @@ protected Flux<HealthResult> aggregateResults(HealthIndicator[] indicators) {
      * @return The aggregated details for the results
      */
     protected Object aggregateDetails(List<HealthResult> results) {
-        Map<String, Object> details = CollectionUtils.newHashMap(results.size());
-        results.forEach(r -> details.put(r.getName(), buildResult(r.getStatus(), r.getDetails(), HealthLevelOfDetail.STATUS_DESCRIPTION_DETAILS)));
-        return details;
+        Map<String, Object> aggregatedDetails = CollectionUtils.newHashMap(results.size());
+        results.forEach(r -> {
+            var name = r.getName();
+            var details = r.getDetails();
+            var status = r.getStatus();
+            aggregatedDetails.put(name, buildResult(status, details, HealthLevelOfDetail.STATUS_DESCRIPTION_DETAILS));
+            if (LOG.isTraceEnabled()) {
+                LOG.trace("Health monitor check for {} with status {}", name, status);
```

Comment:
can we do a single log line instead of 2?

Reply:
yeah, I was just looking at it and thinking the same thing. :)